docs/rules/consistent-test-filename.md

```diff
@@ -1,6 +1,5 @@
 # Require .spec test file pattern (`vitest/consistent-test-filename`)
 
-⚠️ This rule _warns_ in the 🌐 `all` config.
```

Comment:
Is there any specific reason why this was removed? It's generated by a tool we use to keep documentation updated. Did you removed it manually?

Reply:
@veritem 
> Is there any specific reason why this was removed?

Since the information is already documented in the All section, I determined that it is unnecessary to include it in each individual rule's documentation.

>  It's generated by a tool we use to keep documentation updated. Did you removed it manually?

Oh, I see... I didn't realize that and ended up removing it manually. 🥲

Reply:
If you don't mind, you can get them back and I'll go a head and merge your PR

Reply:
I reverted it in commit 62cd790.
Should we add the note "This rule warns in the 🌐 all config." to the documentation for the rules that were enabled this time (e.g., expect-expect, no-commented-out-tests, etc.)?

Reply:
uhmm, it's supposed to be done by `eslint-doc-generator` automatically but it hasn't added support for flat config.

> Should we add the note "This rule warns in the 🌐 all config." to the documentation for the rules that were enabled this time

yes!

Reply:
I think so, for now

Reply:
I added the notes in commit ce2114a.

Reply:
One last thing, did you run this command `update:eslint-docs`? I think some things on readme have been modified which shouldn't be modified.

Reply:
> did you run this command update:eslint-docs?

No, I haven't run it.

> I think some things on readme have been modified which shouldn't be modified.

Which part are you referring to? The [Rules section](https://github.com/y-hsgw/eslint-plugin-vitest/blob/a087a49f30f0254fb196fb52d0057199bf70c385/README.md#rules)?

Reply:
>Which part are you referring to? The [Rules section](https://github.com/y-hsgw/eslint-plugin-vitest/blob/a087a49f30f0254fb196fb52d0057199bf70c385/README.md#rules)?

Yes, the rules section should remain as is.

Reply:
I fixed in 488a269.